test/commands/iteration/create_test.rb

```diff
@@ -21,7 +20,6 @@ class Iteration::CreateTest < ActiveSupport::TestCase
     Iteration::TestRun::Init.expects(:call)
     Iteration::Analysis::Init.expects(:call)
     Iteration::Representation::Init.expects(:call)
-    ToolingJob::Create.expects(:call)
```

Comment:
I had to disable this to get the test passing, but I didn't understand at first why I had to do that. The `Iteration::TestRun::Init` call (and the other two) all call `ToolingJob::Create.` in their respective `.call` methods, but it looks like they were not registering. Turns out that `Iteration::TestRun::Init.expects(:call)` causes this. Is it okay to just remove this expectation?

Reply:
Yeah, I think that's fine now :)